static/app/views/admin/options.tsx

```diff
@@ -49,6 +53,49 @@ const sections: Section[] = [
   },
 ];
 
+const HIGH_THROUGHPUT_RATE_OPTION = {
+  defaultValue: () => '0',
+  component: NumberField,
```

Comment:
Could have used range but it's too easy to let go of the mouse and change an option on a rate.